src/example/example.service.ts

```diff
@@ -0,0 +1,93 @@
+export class ExampleService {
+  public calculateAverage(numbers: number[]): number {
+    if (!numbers.length) return 0;
+    let sum = 0;
+    for (const num of numbers) {
+      sum += num;
+    }
+    return sum / numbers.length;
```

Comment:
PullProofAI

Consider handling the case where `numbers` array is empty in the `calculateAverage` method to avoid division by zero. Returning 0 might not be appropriate in all cases. You could throw an error or return `NaN` to indicate an invalid input.



<!-- This is an auto-generated comment by PullProofAI -->